dev/com.ibm.ws.kernel.service/src/com/ibm/wsspi/kernel/service/utils/SerializableSchedule.java

```diff
@@ -71,123 +84,191 @@ public SerializableSchedule() {
-     * 2. Day(s) and Time(s) are delimited by a space " "
-     *    - Example "MON 8:00" or "MON-FRI 8:00-13:00"
+     * 2. Day(s), time(s), and timezone are delimited by a space " "
+     *    - Example "MON 8:00" or "MON-FRI 8:00-13:00 America/Chicago"
      * 3. A range of days or times are delimited by a hyphen "-"
      *    - Example "MON-FRI 8:00" or "Tues 8:00-13:00"
      * </pre>
      *
-     * Once parsed this object will have the following fields:
-     *
-     * <pre>
-     * - startup: true if scheduled time is not set and instead a flag is set to be checked during server startup, otherwise, false.
-     * - startDay: Represents either the start of a range of days, or a scheduled day.
-     * - endDay: Represents the end of a range of days.
-     * - startTime: Represents either the start of a range of times, or a scheduled time.
-     * - endTime: Represents the end of a range of times.
-     * </pre>
-     *
      * @param strVal - the string to be parsed
      */
     public SerializableSchedule(String strVal) {
+        //Sanitize string "  mon - fri  8:00  -  17:00  GMT+6  " -> "mon-fri 8:00-17:00 GMT+6"
+        String originalStr = strVal;
+        strVal = strVal.trim().replaceAll("( )+", " ").replaceAll("( - )", "-");
+
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "Sanitize Input", originalStr, strVal);
+        }
+
         //Handle STARTUP case
-        if (strVal.trim().equalsIgnoreCase("STARTUP")) {
+        if (strVal.equalsIgnoreCase("STARTUP")) {
             this.startup = true;
             this.validDays = null;
             this.startTime = null;
             this.endTime = null;
+            this.timezone = null;
             return;
         } else {
             this.startup = false;
         }
 
-        //Split MON-FRI 8:00-10:00 to [MON-FRI, 8:00-10:00] OR
-        //Split MON 8:00 to [MON, 8:00]
-        strVal = strVal.trim();
-        String[] splitDaysTimes = strVal.split(" ");
-        if (splitDaysTimes.length != 2) {
-            throw new IllegalArgumentException("Could not parse configuration value as a schedule: " + strVal);
+        //Split "MON-FRI 8:00-10:00 America/Chicago"  to [MON-FRI, 8:00-10:00, America/Chicago]
+        String[] splitDaysTimesZone = strVal.split(" ");
+
+        if (splitDaysTimesZone.length < 2 || splitDaysTimesZone.length > 3) {
+            throw new IllegalArgumentException(Tr.formatMessage(tc, "invalidScheduleDelimit", originalStr, "MON-FRI 8-17:00 GMT-6"));
+        }
+
+        //Parse DayOfWeek
+        //Split "MON-FRI" to [MON, FRI] or "MON" to [MON]
+        String[] splitDays = splitDaysTimesZone[0].split("-");
+
+        // Ensure only one or two values
+        if (splitDays.length < 1 || splitDays.length > 2)
+            throw new IllegalArgumentException(Tr.formatMessage(tc, "invalidScheduleDelimitRange", splitDaysTimesZone[0], "MON-FRI"));
```

Comment:
If someone enters the wrong order and mistakenly uses - instead of _,
`Asia/Ho-Chi-Minh 9:00-17:00 MON-FRI`
then this error message will tell them that they need to use a single hyphen for day ranges, such as MON-FRI and they will suspect that Liberty is at fault instead of their input because they have MON-FRI already.  I think we should at least include the full example with all 3 parts to help them get it right.